test/e2e/run.sh

```diff
@@ -750,7 +783,7 @@ if [ "$reinstall_cri_resmgr" == "1" ]; then
     uninstall cri-resmgr
 fi
 
-if ! vm-command-q "[ -f /usr/local/bin/cri-resmgr ]"; then
+if ! vm-command-q "[ -f /usr/local/bin/cri-resmgr ] || [ -f /usr/bin/cri-resmgr ]"; then
```

Comment:
Wouldn't a `! vm-command-q "[ type -p cri-resmgr ]"` test be better here ?

Reply:
Yes it would! Fixed.